connect-mixin.js

```diff
@@ -18,24 +18,25 @@ subject to an additional IP rights grant found at http://polymer.github.io/PATEN
   class MyElement extends connect(store)(HTMLElement) {
     // ...
 
-    update(state) {
+    stateChanged(state) {
       this.count = state.data.count;
     }
   }
 */
 
 export const connect = (store) => (baseElement) => class extends baseElement {
-  constructor() {
-    super();
-
+  connectedCallback() {
     // Connect the element to the store.
-    store.subscribe(() => this.update(store.getState()));
-    this.update(store.getState());
+    this.__storeUnsubscribe = store.subscribe(() => this.stateChanged(store.getState()));
+    this.stateChanged(store.getState());
```

Comment:
Missing `super.connectedCallback();`. Also let's add the `super` at the end of the method so the flow is `stateChanged` -> `ready` -> `render` when extending from `LitElement`

Reply:
done!